src/mon/PGMap.cc

```diff
@@ -813,6 +816,39 @@ void PGMapDigest::dump_pool_stats_full(
   }
 }
 
+void PGMapDigest::dump_meta_pool_stats(
+  stringstream* ss,
+  ceph::Formatter* f) const
+{
+  const pool_stat_t* meta_pool_stats = nullptr;
+  auto it = pg_pool_sum.find(-1);
+  if (it != pg_pool_sum.end()) {
+    meta_pool_stats = &(it->second);
```

Comment:
nit: a `const`-reference defined here instead of the pointer above?

Reply:
Nope. This wouldn't cover the case when find(-1) gets no entry.